src/services/userList.ts

```diff
@@ -27,4 +28,19 @@ export interface userListParam {
       throw error;
     }
   };
+  
+
+  export const getCohortList = async (
+    userId: string | string[],
+   
```

Comment:
@AkshataKatwal16 create separate service for the cohorts and add this function there